extensions/amp-image-slider/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,36 @@
+import {text, withKnobs} from '@storybook/addon-knobs';
+
+import * as Preact from '#preact';
+
+import {BentoImageSlider} from '../component';
+
+export default {
+  title: 'ImageSlider',
+  component: BentoImageSlider,
+  decorators: [withKnobs],
+};
+
+export const _default = () => {
+  const first = text(
+    'First image',
+    'https://amp.dev/static/samples/img/canoe_900x600.jpg'
+  );
+  const second = text(
+    'Second image',
+    'https://amp.dev/static/samples/img/canoe_900x600_blur.jpg'
+  );
+
+  return (
+    <BentoImageSlider
+      layout="responsive"
+      width="100"
+      height="200"
+      initial-slider-position="0"
+    >
+      <amp-img slot="first-image" src={first} alt="A green apple"></amp-img>
+      <amp-img slot="second-image" src={second} alt="A red apple"></amp-img>
+      <div slot="first-label">Clear Picture</div>
+      <div slot="second-label">Blur Picture</div>
+    </BentoImageSlider>
```

Comment:
The Preact component API should be different from the AMP one:
```suggestion
    <BentoImageSlider
      initialSliderPosition={0}
      firstImageAs={(props) => (
        <img src={first} alt="A green apple" {...props} />
      )}
      firstLabelAs={(props) => <div {...props}>Clear picture</div>}
      secondImageAs={(props) => (
        <img src={second} alt="A red apple" {...props} />
      )}
      secondLabelAs={(props) => <div {...props}>Blur picture</div>}
      style={{width: 100, height: 200}}
    ></BentoImageSlider>
```